### DISCUSSION TITLE
Is it possible to map db to a network drive ?

### DISCUSSION THREAD

**Discussion by sandraplato1234** [milvus-io/milvus#5637] [Q&A and General discussion]
Hi 
Is it possible to map db to a network drive?ance
I tried thrugh docker run -v parameter but it seems that milvus does not started propely
it stucks after writing waring you are using SQLite

Thanks in advance

**Comment by yhmo** [comment]
The warning for SQLite doesn't matter. Do you start milvus by docker? To mapping db folder to other path, you need to do the following things:
1. download a config file and put into a local folder:
```
mkdir -p /xxx/milvus/conf
cd /xxx/milvus/conf
wget https://raw.githubusercontent.com/milvus-io/milvus/v1.1.0/core/conf/demo/server_config.yaml
```

2. start milvus server by container, mapping internal path to outside path:
docker run -it -p 19530:19530 -v /xxx/milvus/db:/var/lib/milvus/db -v /xxx/milvus/conf:/var/lib/milvus/conf -v /xxx/milvus/logs:/var/lib/milvus/logs milvusdb/milvus:1.1.0-cpu-d050721-5e559c

**Reply by sandraplato1234** [reply]
Ok thanks that's I did but instead of an empty db I copy to  milvus db folder a db from another instance of milvus that has already collections and got stuck after that
When the folder of db is empty it's works fine

**Comment by yhmo** [comment]
Milvus has two kinds of data: 
1. vectors/index data stored in db folder
2. metadata stored in sqlite or mysql, metadata is information of collections/segments, such as collection name/dimension, segment row_count/file_size/index_type, etc.

SQLite metadata is stored under the db folder, so if the "another instance of milvus" use sqlite, only need to copy the db folder for new milvus instance.
But if the "another instance of milvus" use mysql, you need to start a mysql server, import metadata from the mysql of another milvus instance. 

When milvus start, the console print something like:

```
   /  |/  /  _/ /| | / / / / / __/    
  / /|_/ // // /_| |/ / /_/ /\ \    
 /_/  /_/___/____/___/\____/___/     

Welcome to use Milvus!
Milvus Debug version: v1.1.1, built at 2021-06-09 17:25.19, with OpenBLAS library.
You are using Milvus GPU edition
Last commit id: 5386d7ddb7b7e1e2048ee100f50df8fac79fbb32

Loading configuration from: /home/yhmo/git/milvus/core/conf/server_config.yaml
GPU resources ENABLED !
NOTICE: You are using SQLite as the meta data management, We recommend change it to MySQL.
Supported CPU instruction sets: avx2, sse4_2
Nvidia driver version: 450.57
CUDA Driver Version / Runtime Version : 11.0 / 10.2
FAISS hook AVX2
Milvus server started successfully!
```

For "it stucks after writing waring you are using SQLite", could you show me what it print on the console?

**Reply by sandraplato1234** [reply]
Hi,
The last sentence written to the console is:
**FAISS hook AVX2**

**Reply by yhmo** [reply]
Could you show me the xxx.debug.log when it stuck.
